components/common/Button.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { Link } from 'routes';
+
+export default function Button(props) {
+  const classes = ['c-button'];
+
+  // We add the classes
+  if (props.secondary) classes.push('-secondary');
+  else classes.push('-primary');
+
+  if (props.inverse) classes.push('-inverse');
+
+  if (props.link) {
+    // If the link is a string, this means that it is external
+    const isExternalLink = typeof props.link === 'string';
+
+    const linkAttributes = isExternalLink
```

Comment:
I don't think that can be extracted. It's very specific to the component (because of the props).